ql/src/test/results/clientnegative/udf_assert_true.q.out

```diff
@@ -21,29 +21,17 @@ STAGE PLANS:
       Processor Tree:
         TableScan
           alias: src
-          Lateral View Forward
-            Select Operator
-              Lateral View Join Operator
-                outputColumnNames: _col6
-                Limit
-                  Number of rows: 2
-                  Select Operator
-                    expressions: assert_true((_col6 > 0)) (type: void)
-                    outputColumnNames: _col0
-                    ListSink
-            Select Operator
-              expressions: array(1,2) (type: array<int>)
-              outputColumnNames: _col0
-              UDTF Operator
-                function name: explode
-                Lateral View Join Operator
-                  outputColumnNames: _col6
-                  Limit
-                    Number of rows: 2
-                    Select Operator
-                      expressions: assert_true((_col6 > 0)) (type: void)
-                      outputColumnNames: _col0
-                      ListSink
+          Select Operator
```

Comment:
Yes.  I mentioned this in the comment.  Basically, this can happen when the select above the lateral view only contains columns that are produced by the UDTF.  In that case, there is no need to do the lateral join back to the base table.